lib/donations/currencies.rb

```diff
@@ -0,0 +1,47 @@
+# == Currency Converter to JSON
+#
+# Takes an array of integer amounts, in US cents, and
+# converts to GBP and EUR.
+#
+# ==== Examples
+#
+#   currencies = Donations::Currencies.for(102, 204, 302, 401)
+#   currencies.to_hash # also to_json
+#   =>
+#   {
+#     USD: [1.02, 2.04, 3.02, 4.01],
```

Comment:
That sounds good. @osahyoun I agree it makes sense for that to live on the server. I'll just assume on the JS for now that it should just display the array of values.